### PR TITLE
[T3A3][F11-A2] Le Minh Duc 

### PR DESCRIPTION
**Pull request by darrenlmd**

### CODE REVIEW

**Comment by acuodancer** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -26,6 +32,26 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
...
+    
+    /**
+     * 
+     * @param address
+     */
+    private void decodeAddress(String address) {
+    	// credit use of split to user872858 at http://stackoverflow.com/questions/10631715/how-to-split-a-comma-separated-string
+    	String[] addressElements = address.split("\\s*,\\s*");
+    	
+    	Block newBlock = new Block(addressElements[0]);
```
Comment (Candiie) [inline src/seedu/addressbook/data/person/Address.java]: Hi Duc, do try to remove magic numbers like this :)

**Comment by Candiie** [comment]
Some comments added. Please close the PR after reading comments :+1: Cleanly executed, the significance per commit is done really well, good job. I do appreciate the consistent commit messages too ! Very Well done!

**Comment by Candiie** [comment]
Helloooo dear Duc, do review my comments and close this PR. Do note that if you do not close more then 2 weeks worth of PR I might just stop reviewing your work until you've reviewed my comments & close the current PRs ~ Candiie

**Pull request closed without merging**